packages/ngx-forms/src/lib/datepicker/components/datepicker/datepicker.component.ts

```diff
@@ -102,12 +104,22 @@ export class DatepickerComponent implements OnInit, OnDestroy, ControlValueAcces
             this.onChange(value);
           }
         } else {
-          this.selectedDate = null;
+          this.selectedDate = this.min;
```

Comment:
@jsebrech, I understand your concerns. But it seems logic to me that if **there is a min property** defined in the datepicker and **there isn't a current value** that the first selectable date is the min date. Although, this has to be the case for our application.

I agree about the edge case that if the current date is in the interval it's better to set the date to the current date.

Reply:
Yes, the first selectable date should be the min date, but it should not be automatically selected. When I try out this code the min automatically appears as selected, even when no date has been selected yet. Moving this change into calendar.component.ts would solve this, because there it is possible to set activeDate to the interval.min when selectedDate is null.